src/modules/frontpage/frontpageApi.js

```diff
@@ -24,3 +25,26 @@ export const updateFilmFrontpage = filmfrontpage => {
     body: JSON.stringify(filmfrontpage),
   }).then(resolveJsonOrRejectWithError);
 };
+
+export const fetchSubjectpage = (id, language) => {
+  const query = queryString.stringify({ language });
+  const url = `http://localhost:1339/frontpage-api/v1/subjectpage/${id}`;
+  const urlLang = language ? url + `?${query}&fallback=true` : url;
+  return fetchAuthorized(urlLang).then(resolveJsonOrRejectWithError);
+};
+
+export const updateSubjectpage = (subjectpage, subjectpageId) => {
+  return fetchAuthorized(
+    `http://localhost:1339/frontpage-api/v1/subjectpage/${subjectpageId}`,
```

Comment:
Her må du legge inn url avhengig av miljø. Og du kan deploye frontpage-api til test fra branch om du ønsker for testing.